### DISCUSSION TITLE
cantina contest

### DISCUSSION THREAD

**Discussion by sagoto80** [Cyfrin/security-and-auditing-full-course-s23#82] [Q&A]
How i can get invitation code for register .
isnot it open for everyone

**Comment by johnsonstephan** [comment]
Create an account at http://cantina.xyz and you can participate. 

I assume you are looking at the recent Superform announcement -- details here: https://cantina.xyz/competitions/f0cfe0e9-290b-4967-94ac-504375329127.

Here's the twitter post - https://x.com/cantinaxyz/status/1740496509752537091?s=20


<details><summary>Bonus</summary>
<p>

Th $27,500 Stake.link contest on CodeHawks competition is ending in a couple days and you can register here: https://www.codehawks.com/contests/clqf7mgla0001yeyfah59c674

</p>
</details> 
---

Does this answer your question?

**Reply by johnsonstephan** [reply]
@sagoto80 Were you able to resolve this?

**Reply by sagoto80** [reply]
nope. it require invitation code
